tests/moead_test.cpp

```diff
@@ -498,3 +496,48 @@ TEST_CASE("MOEADFonsecaFlemingTestVectorFloatBounds", "[MOEADTest]")
 
   REQUIRE(allInRange);
 }
+
+// /**
```

Comment:
Any reason why that one is commented out?

Reply:
No reason! I was just testing against NSGA-II. I'll revert that be assured.